xml/art_admin_slemicro.xml

```diff
@@ -1207,6 +1207,82 @@ After=etcd.service
     </variablelist>
    </section>
   </section>
+  <!-- Move this section to a separate Cockpit guide -->
+  <section xml:id="sec-cockpit-updates">
+   <title>Managing &slema; updates and snapshots</title>
+   <note>
+    <title>Snapshots and updates management only for system administrators</title>
+    <para>
+     Only users with the <guimenu>Server administrator</guimenu> role can
+     update the system or perform a rollback to another snapshot.
+    </para>
+   </note>
+   <para>
+    Cockpit enables you to update your &slema; instance or perform a rollback
+    from the <literal>Software Updates</literal> menu.
+   </para>
+   <figure>
+    <title>Software Updates</title>
+    <mediaobject>
+     <imageobject role="fo">
+      <imagedata fileref="cockpit_software_updates.png" width="100%"/>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="cockpit_software_updates.png" width="100%"/>
+     </imageobject>
+    </mediaobject>
+   </figure>
+   <section xml:id="sec-cockpit-updates-procedure">
+    <title>Managing updates</title>
+    <para>
+     Click <guimenu>Check for Updates</guimenu> to get a list of new packages
+     updates and patches available for your system. You can view each package
+     update or patch information by clicking the blue icon next to the package
+     update/patch. It is recommended to install the patches marked as important
+     as soon as possible.
+    </para>
+    <para>
+     To apply the updates and patches, click <guimenu>Update and
+     Reboot</guimenu>. The command <command>transactional-update up</command>
```

Comment:
The actual work is done via DBUS API so to be 100% correct I would say something like "The equivalent of `transactional-update up` command is called..."